ic-stable-structures/src/memory_utils.rs

```diff
@@ -0,0 +1,23 @@
+use std::thread::LocalKey;
+
+use dfinity_stable_structures::{
+    memory_manager::{MemoryId, MemoryManager, VirtualMemory},
+    Memory,
+};
+
+#[cfg(all(not(target_arch = "wasm32")))]
+pub type DefaultMemoryResourceType = dfinity_stable_structures::VectorMemory;
+
+#[cfg(target_arch = "wasm32")]
+pub type DefaultMemoryResourceType = dfinity_stable_structures::Ic0StableMemory;
+
+pub type DefaultMemoryType = VirtualMemory<DefaultMemoryResourceType>;
+pub type DefaultMemoryManager = MemoryManager<DefaultMemoryResourceType>;
```

Comment:
I remember we decided not to have a default memory manager

Reply:
Yes, but in the most of the canisters we use vector memory for native (tests) and `IC0StableStorage` for canister. So having the type aliases is super handy.
I've removed the memory manager instance from here so we are not paying anything in runtime, just useful aliases and the client decided to use them or not.